packages/react/ds/src/forms/form-field/form-field.tsx

```diff
@@ -81,17 +86,27 @@ const FormFieldBase = ({
   const allChildren = Children.toArray(children);
 
```

Comment:
alt:

`(child.type as any).componentType || (child?.props as any)?.__mdxType`

maybe assigning to a type variable like:

`const type = (child.type as any).componentType || (child?.props as any)?.__mdxType;`

then you can just compare `type === 'FormFieldLabel'` (and so on) for each case, cleaner and easier to follow.